.editorconfig

```diff
@@ -13,3 +13,6 @@ trim_trailing_whitespace = false
 
 [*.{yml,yaml}]
 indent_size = 2
+
+[docker-compose.yml]
+indent_size = 4
```

Comment:
Can be the same as for all `.yml` files (see the entry above)

Reply:
copy pasta kicks in. fixed

Reply:
Thanks! :)